syntax = "proto3";

package grpc.tradeapi.v1;

option csharp_namespace = "Finam.TradeApi.Grpc.V1";

import "finam_grpc_client/models/proto/candles.proto";

service Candles {
  // Returns the list of day candles.
  // Возвращает дневные свечи.
  rpc GetDayCandles(proto.tradeapi.v1.GetDayCandlesRequest) returns (proto.tradeapi.v1.GetDayCandlesResult);

  // Returns the list of intraday candles.
  // Возвращает внутридневные свечи.
  rpc GetIntradayCandles(proto.tradeapi.v1.GetIntradayCandlesRequest) returns (proto.tradeapi.v1.GetIntradayCandlesResult);
}